apps/framework-landing/app/sections/FooterSection.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  FullWidthContentContainer,
+  Section,
+} from "@/components/containers/page-containers";
+
+import Link from "next/link";
+import { cn } from "@/lib/utils";
+import { SmallText } from "@/components/typography/standard";
+
+export const FooterSection = () => {
+  return (
+    <Section>
+      <FullWidthContentContainer className="">
+        <FooterContent />
+      </FullWidthContentContainer>
+    </Section>
+  );
+};
+
+export const FooterNavItem = ({
+  item,
+  children,
+  className,
+}: {
+  item: { name: string; href: string };
+  children: string;
+  className?: string;
+}) => {
+  return (
+    <Link href={item.href} className={cn("text-foreground", className)}>
+      <SmallText> {children} </SmallText>
+    </Link>
+  );
+};
+
+export const FooterNav = () => {
+  const navigation = [
```

Comment:
This looks like could be reused with the `Nav` components if we want to keep things consistent

Reply:
No, they are different navs

Reply:
As in they behave very differently